manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -815,12 +815,16 @@ func (r *FybrikApplicationReconciler) buildSolution(applicationContext Applicati
 		ProvisionedStorage: make(map[string]NewAssetInfo),
 	}
 
+	modulesNamespace := utils.GetFybrikModuleNamespaceFromLabels(applicationContext.Application.GetLabels())
+	if modulesNamespace == "" {
+		modulesNamespace = utils.GetDefaultModulesNamespace()
+	}
```

Comment:
1. This code appears in several places and can be replaced by a single function.
2. We can deploy Fybrik in two ways: `.Values.modulesNamespace.managedByFybrik` = true or false. If it is true, the function should ignore this label and work as previously.